maquetado/setup/index.html

```diff
@@ -0,0 +1,14 @@
+<!doctype html>
+  <html lang="en">
```

Comment:
This is ok, but the `doctype` tag and the `html` should be in the same level, it is not a containing tag